File: DRRR.Server/Dtos/UserRegisterRequestDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace DRRR.Server.Dtos
{
    /// <summary>
    /// 用户注册请求DTO
    /// </summary>
    public class UserRegisterRequestDto
    {
        /// <summary>
        /// 用户名
        /// </summary>
        public string Username { get; set; }

        /// <summary>
        /// 密码
        /// </summary>
        public string Password { get; set; }

        /// <summary>
        /// 验证码
        /// </summary>
        public string Captcha { get; set; }

        /// <summary>
        /// 验证码哈希值
        /// </summary>
        public string CaptchaHash { get; set; }
    }
}

File: DRRR.Client/src/app/user/dtos/user-register-request.dto.ts
export interface UserRegisterRequestDto {
  /**
   * 用户名
   */
  username: string;

  /**
   * 密码
   */
  password: string;

  /**
   * 验证码ID
   */
  captchaId: string;

  /**
   * 验证码文本
   */
  captchaText: string;
}
